### DISCUSSION TITLE
Entendimento da extensão

### DISCUSSION THREAD

**Discussion by vinijmoura** [cribeiro84/azure-devops-pull-request-hub#196] [Q&A]
Ola Carlos, bom dia,

Queria falar contigo a respeito da extensão. Se possivel voce pode me mandar um e-mail por favor.

vinijmoura@hotmail.com

Obrigado,

Vinicius Moura.

**Comment by cribeiro84** [comment]
Olá Vinicius, bom dia.
Sobre o que seria exatamente? Se puder adiantar por aqui.

Muito obrigado.

**Reply by vinijmoura** [reply]
Eu uso muito sua extensao e recomendo pra muita gente. Parabens!
Eu escrevo muito sobre Azure DevOps (principalmente sobre parte de relatorios). E queria usar sua extensao como exemplo para criar um Dashboard em Power BI.
Da parte de PR, qual API voce está usando por favor?

**Reply by cribeiro84** [reply]
Opa, que bacana! Agradeço pelo feedback também.
Sobre a API, uso as opções internas do próprio SDK do Azure DevOps APIs.

esse aqui:



`await this.gitClient.getPullRequests(          r.id,          criteria,          undefined,          10,          undefined,          top        );`

**Reply by vinijmoura** [reply]
show de bola! vou buscar o correspondente na REST para chamar no Power BI. Vou evoluir no relatorio. Ao publicar o post, vou citar sua extensao e te citar no twitter.

Muito obrigado =)

**Reply by cribeiro84** [reply]
No DOCS da Microsoft tem toda a especificação de como usar:

https://docs.microsoft.com/en-us/rest/api/azure/devops/?view=azure-devops-rest-6.1

:-)

**Reply by vinijmoura** [reply]
Ja consegui pegar todas as APIs. Vou montar o relatorio :-)
Obrigado pela ajuda!
